jsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+    "compilerOptions": {
+        "module": "ESNext",
+        "moduleResolution": "Bundler",
+        "checkJs": true,
+        "target": "ES2022",
+        "allowImportingTsExtensions": true,
+        "strictNullChecks": true,
+        "strictFunctionTypes": true
+    },
+    "exclude": [
+        "node_modules",
+        "**/node_modules/*"
+    ]
+}
```

Comment:
Dev:
	This gives typechecking in VSCode Studio (it uses the typescript typechecker).